calyx-py/calyx/queue_call.py

```diff
@@ -139,7 +136,9 @@ def insert_main(
     stats_component=None,
 ):
     """Inserts the component `main` into the program.
-    It triggers the dataplane and controller components.
+    It triggers the dataplane and controller components. If there is an error,
+    the value 20000 is used as the value. If there is a push, the value 10000
+    is used as the value.
```

Comment:
Pretty sure some of your changes from the interface-tidying has leaked into this PR. No worries though, let's just power through